app/models/post.rb

```diff
@@ -3,6 +3,12 @@ class Post < ApplicationRecord
   has_many :likes
   has_many :comments
 
+  after_save :post_counter
+
+  validates :title, presence: true, length: { in: 0..250 }
```

Comment:
- [ ] **Optional:** The length should extend probably from 3 to 250 as there is hardly a title of 2 or 3 characters that is truly meaningful. 😏